src/main/java/com/zaxxer/hikari/pool/PoolElf.java

```diff
@@ -219,7 +219,9 @@ boolean isConnectionAlive(final Connection connection, AtomicReference<Throwable
          final int originalTimeout = getAndSetNetworkTimeout(connection, validationTimeout);
    
          try (Statement statement = connection.createStatement()) {
-            setQueryTimeout(statement, timeoutSec);
+        	if (isNetworkTimeoutSupported != TRUE) {
+               setQueryTimeout(statement, timeoutSec);
```

Comment:
Maybe github is showing it wrong.  The 'if' and the 'setQueryTimeout()' appear at the same indentation in the pull request on their site.